documentation/meta/maintenance/elasticsearch_cluster.md

```diff
@@ -394,7 +401,7 @@ or "production".
    pass the public IPv4 DNS step from step 3 to the `-l`/`--limit` flag.
 
    ```bash
-   just ansible/playbook <env> elasticsearch/sync_config.yml -e apply=true -l <public_ipv4_dns>`
+   just ansible/playbook <environment> elasticsearch/sync_config.yml -e apply=true -l <public_ipv4_dns>
```

Comment:
Might be a good idea to provide the options for environment (`<environment(prod/dev)>`, or is it `<environment(production/development)>`)